### PR TITLE
add swapline collateral value to TVL

### PR DESCRIPTION
**Pull request by PawelKrawczyk96**

### CODE REVIEW

File: src/store/selectors/exchange.ts [hunk 1]
```diff
@@ -240,6 +240,23 @@ export const exchangeSelectors = {
...
+  swaplines,
+  collaterals,
+  assets,
+  (allSwaplines, allCollaterals, allAssets) => {
+    const balanceUSD = Object.values(allSwaplines).map((swapline) => {
+      const collateral = allCollaterals[swapline.collateral.toString()]
+      if (!collateral) {
+        return 0
+      } else {
+        return (+printBN(swapline.balance.val, swapline.balance.scale) - +printBN(swapline.accumulatedFee.val, swapline.accumulatedFee.scale)) * +printBN(allAssets[collateral.assetIndex].price.val, allAssets[collateral.assetIndex].price.scale)
```
Comment (awojciak) [inline src/store/selectors/exchange.ts:253]: maybe "swapline.balance.val.sub(swapline.accumulatedFee.val).mul(allAssets[collateral.assetIndex].price.val).div(swapline.balance.scale).div(allAssets[collateral.assetIndex].price.scale)" ?

File: src/containers/Statistics/StatisticsCard.tsx [hunk 2]
```diff
@@ -3,11 +3,12 @@ import React from 'react'
 import { useSelector } from 'react-redux'
 
 import statsSelector from '@selectors/stats'
-import { getCollateralValue, getSyntheticsStructure } from '@selectors/exchange'
+import { getCollateralValue, getSyntheticsStructure, getSwaplineCollateralBalance } from '@selectors/exchange'
 
 export const StatisticsCard: React.FC = () => {
   const statsData = useSelector(statsSelector.last24)
   const synthetics = useSelector(getSyntheticsStructure)
+  const USDBalance = useSelector(getSwaplineCollateralBalance)
```
Comment (awojciak) [inline src/containers/Statistics/StatisticsCard.tsx:11]: maybe swaplineCollateralValue instaed of USDBalance?

Comment (PawelKrawczyk96) [reply src/containers/Statistics/StatisticsCard.tsx:11]: okay, make sense

File: src/components/Statistics/StatisticCardAll.tsx [hunk 3]
```diff
@@ -23,6 +23,7 @@ export const StatisticCardAll: React.FC<IProp> = ({ data, debtCurrent, collatera
   return (
     <div className={classes.gridContainer}>
       <Grid container className={classes.container}>
+        {console.log(debtCurrent)}
```
Comment (wojciech-cichocki) [inline src/components/Statistics/StatisticCardAll.tsx:26]: `console.log` to remove

**Review by wojciech-cichocki — APPROVED** [review]

**Review by NorbertBodziony — APPROVED** [review]

**Pull request merged**